NucleoTools/modules/dopskript.py

```diff
@@ -0,0 +1,40 @@
+def run_dna_rna_tools(*args):
+    *seqs, action = args
+    if action == "transcribe":
+        result = transcribe(seqs)
+        return "".join(result) if len(seqs) == 1 else result
+    elif action == "reverse":
+        result = reverse(seqs)
+        return "".join(result) if len(seqs) == 1 else result
+    elif action == "complement":
+        result = complement(seqs)
+        return "".join(result) if len(seqs) == 1 else result
+    elif action == "reverse_complement":
+        result = reverse_complement(seqs)
+        return "".join(result) if len(seqs) == 1 else result

```

Comment:
```suggestion
def run_dna_rna_tools(*args):
    *seqs, action = args
    result = []   
    if action == "transcribe_mine":
        action = transcribe_mine
    elif action == "reverse":
        action = reverse
    elif action == "complement":
        action = complement
    elif action == "reverse_complement":
        action = reverse_complement
    if len(seqs) == 1:
        return action(seqs[0])
    for seq in seqs:
        result.append(action(seq))
    return result
```
При этом я бы оставил все эти условия где `action == 'action' `
Но проверку бы на len сделал бы одну в начале
Но побольшей части это все равно придирки, так как в любом случае твой код все равно хорошо работает

Reply:
А еще не хватает функции проверки сиквенса, а то я как будто туда могу и `"dkofpavalpcd[fqe"` кинуть